# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrtools/no-junk.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


We neither want Solaris or commercial flavour advertising, nor needless
delays and everything is a /dev file bitching ...

  - Rebe Rebe <rene@exactcode.de>

diff -r -U 1 cdrtools-2.01/cdrecord/cdrecord.c cdrtools-2.01-nonverb/cdrecord/cdrecord.c
--- cdrtools-2.01/cdrecord/cdrecord.c	2004-09-08 19:26:35.000000000 +0200
+++ cdrtools-2.01-nonverb/cdrecord/cdrecord.c	2006-01-13 16:06:23.000000000 +0100
@@ -666,5 +666,3 @@
 			errmsgno(EX_BAD,
-			"If you need DVD-R/DVD-RW support, ask the Author for cdrecord-ProDVD.\n");
-			errmsgno(EX_BAD,
-			"Free test versions and free keys for personal use are at ftp://ftp.berlios.de/pub/cdrecord/ProDVD/\n");
+			"Try dvd+rwtools instead.\n");
 		}
@@ -3584,4 +3582,6 @@
 			errmsgno(EX_BAD, "Future versions of cdrecord may have different drive dependent defaults.\n");
+#if 0
 			errmsgno(EX_BAD, "Continuing in 5 seconds...\n");
 			sleep(5);
+#endif
 			tao = 1;
@@ -4680,3 +4680,3 @@
 {
-#if	defined(linux) || defined(__linux) || defined(__linux__)
+#if	0
 #ifdef	HAVE_UNAME
diff -r -U 1 cdrtools-2.01/libscg/scsi-linux-sg.c cdrtools-2.01-nonverb/libscg/scsi-linux-sg.c
--- cdrtools-2.01/libscg/scsi-linux-sg.c	2004-05-20 15:42:12.000000000 +0200
+++ cdrtools-2.01-nonverb/libscg/scsi-linux-sg.c	2006-01-13 15:26:47.000000000 +0100
@@ -499,2 +499,3 @@
 		}
+#if 0
 		if (scgp->overbose) {
@@ -512,2 +513,3 @@
 		}
+#endif
 					/* O_NONBLOCK is dangerous */
